charts/db-backup/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for db-backup.
+
+nameOverride: ""
+fullnameOverride: ""
+# https://github.com/alphagov/govuk-infrastructure/tree/main/images/toolbox
+imageRef: "172025368201.dkr.ecr.eu-west-1.amazonaws.com/toolbox"
+
+# govukEnvironment determines which set of cronjobs to use from `cronjobs`
+# below. When running under Argo CD, the ../app-config chart passes the
+# appropriate value for govukEnvironment in by default.
+govukEnvironment: staging
+
+# cronjobs defines a set of k8s CronJobs that should exist in each environment
+# (production, staging, integration). Each CronJob consists of one or more
+# operations which run as a sequence of InitContainers.
+#
+# Each environment has a map of job names to job configs. If the job name (i.e.
+# the map key) is of the form <app-name>-<dbms>, for example
+# link-checker-api-postgres, then the DBMS hostname and DBMS type (postgres,
+# mysql etc.) can be inferred. These can be specified explicitly via the `host`
+# and `dbms` keys in the job config.
+cronjobs:
+  production:
+    content-store-postgres:
+      schedule: "16 1 * * *"
```

Comment:
I think this timing could be a bit too tight - if I'm reading it correctly, the mongo-to-postgres cron job starts at 00:16, and the backup of content-store-postgres starts an hour later at 01:16 . But the mongo-to-postgres cron job _currently_ takes about 1hr 15mins to run - [example from prod](https://argo.eks.production.govuk.digital/applications/govuk-jobs?orphaned=false&resource=&node=%2FPod%2Fapps%2Fcontent-store-mongo-to-postgres-cron-28274790-gsdfk&tab=summary):  

```
          finishedAt: '2023-10-05T07:56:31Z'
          reason: Completed
          startedAt: '2023-10-05T06:41:22Z'
```

Reply:
Ah good catch, thanks!